{"name": "test_006", "status": "skipped", "statusDetails": {"message": "XFAIL\n\nAssertionError: assert 21000 == 200", "trace": "@pytest.mark.xfail\n    def test_006():\n        a = 1000\n        b = 20000\n        result = a + b\n    \n>       assert result == 200\nE       assert 21000 == 200\n\nSeleniumProgram\\test_markers.py:60: AssertionError"}, "start": 1673446087894, "stop": 1673446087894, "uuid": "97fe51bc-4de6-4d68-a13d-0eea1b95bb9c", "historyId": "c9c0cf603276bc2bb4b1f9dd5a061d76", "testCaseId": "930fd3ee53eb37cbe085979f22e88ac6", "fullName": "test_markers#test_006", "labels": [{"name": "tag", "value": "@pytest.mark.xfail"}, {"name": "suite", "value": "test_markers"}, {"name": "host", "value": "HP"}, {"name": "thread", "value": "33596-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_markers"}]}